lib/Basics/threads-posix.cpp

```diff
@@ -103,7 +103,29 @@ bool TRI_StartThread(TRI_thread_t* thread, TRI_tid_t* threadId,
 
   TRI_ASSERT(d != nullptr);
 
-  int rc = pthread_create(thread, nullptr, &ThreadStarter, d.get());
+  pthread_attr_t 	stackSizeAttribute;
+  size_t			stackSize = 0;
+
+  auto err = pthread_attr_init (&stackSizeAttribute);
+  if (err) {
+    // We should never be here
+    TRI_ASSERT(false);
```

Comment:
Can we add a proper exception here please.
This will fail in our tests, but in delivered versions if it shows up (even if it is super unlikely obviously it is still possible) we will end up in undefined behaviour otherwise.

Reply:
Good point. I also considered this as ultra unlikely and

Reply:
Done.